### ISSUE TITLE
Automagic review of URLs

### ISSUE THREAD

**Issue by wwarriner** [uabrc/uabrc.github.io#9]
Automatic review of URLs within the docs to update if they are broken
Talk with Louis about continuous integration
Output to log or other file descriptor as necessary instead of just a bare print() call.

**Comment by wwarriner** [comment]
https://pypi.org/project/mkdocs-linkcheck/

**Comment by wwarriner** [comment]
Looks like a form of this is built in to `mkdocs build` with the `-s/--strict` flag.

```
  -s, --strict                    Enable strict mode. This will cause MkDocs
                                  to abort the build on any warnings.
```

I'll put a pull request in for it today. The PR will take care of any currently-broken links. I will also make notes in the contributor guide so we don't end up breaking the build.

**Comment by wwarriner** [comment]
Will also fix #239
